### DISCUSSION TITLE
I think I found a new exploit to enter guest mode on a chromebook that has it blocked.

### DISCUSSION THREAD

**Discussion by no-name-6655** [3kh0/ext-remover#1247] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Patched on v130+

As the title says I think I found one. The only requirement is a WiFi network with a captive login portal. I don't know if this is the right place to share it or is there someone I message? I am not very familiar with GitHub.  
Also, the exploit is useful because entering guest mode removes all extensions and policies, so no site blocking and you can easily get around network filters with the network settings. But you cannot download extensions, use any bookmarks, or download any apps.

**Comment by Balistic123** [comment]
can you provide?

**Reply by no-name-6655** [reply]
Step 1: Powerwash the chromebook
Step 2: During setup select a wifi network that opens a captive portal
Step 3: The chromebook usually gets stuck on a loading screen for a few seconds up to 5 mins. Just wait
Step 4: Wait for a captive portal popup that has a URL bar at the top. It has to be the one with the URL bar.
Step 5: Click the URL once to highlight it, then right click it once. This should make the screen turn black for a few seconds (I have no idea why?)
Step 6: Close the portal and on the network list there should be a little link that will open guest mode
![unnamed](https://github.com/user-attachments/assets/5c0ba7a7-e0ee-4324-8b37-964c1b564162)
![unnamed (1)](https://github.com/user-attachments/assets/7d944747-5d40-4c50-ad98-af8f25054f79)

If publishing makes it so that google will patch this (do they watch this repo?). then I ask that you don't publish it.
Works on all versions I'm mostly certain

**Reply by Sam-hacks1** [reply]
ima try it

**Reply by Sam-hacks1** [reply]
it works but you can't easily go back

**Reply by jazzymakesstuff** [reply]
im pretty sure im still able to access this window

**Reply by jazzymakesstuff** [reply]
my friend found a (maybe?) alternate way to powerwash and this pops up sometimes

**Reply by akabutnicer** [reply]
elaborate and which version was this allegedly found on?

**Reply by jazzymakesstuff** [reply]
actually nvm it's the normal method I'm retarded

**Reply by jazzymakesstuff** [reply]
but I do recall it being there perhaps maybe (131)

**Reply by jazzymakesstuff** [reply]
take everything I say with a grain of salt my friend does weird incomprehensible shit on his computer

**Reply by no-name-6655** [reply]
> my friend found a (maybe?) alternate way to powerwash and this pops up sometimes

the powerwash method wouldn't matter at all, idkwym

**Reply by SlitheringLizard1** [reply]
idk where I could find a captive portal wifi

(I don't even know what a captive portal is lol)

**Reply by no-name-6655** [reply]
A captive portal is a pop-up that appears on a sign-in required network. Usually a public free guest wifi will require sign-in, and most schools have a guest wifi.

**Reply by SlitheringLizard1** [reply]
hm interesting

**Reply by POX-dev** [reply]
guys you can make your own captive portal by changing your nameservers on your home wifi network to 150.136.163.0

**Reply by ghost** [reply]
basically icognito

**Comment by ghost** [comment]
smart :)

**Comment by ghost** [comment]
i had a sucessor https://github.com/coder0-code/dunroll

**Comment by akabutnicer** [comment]
> As the title says I think I found one. The only requirement is a WiFi network with a captive login portal. I don't know if this is the right place to share it or is there someone I message? I am not very familiar with GitHub. Also, the exploit is useful because entering guest mode removes all extensions and policies, so no site blocking and you can easily get around network filters with the network settings. But you cannot download extensions, use any bookmarks, or download any apps.

what version was this found on?

**Reply by no-name-6655** [reply]
127

**Comment by 88411** [comment]
Not worth it cause you have to powerwash everytime. 
This method is better if you have goguardian :
https://youtu.be/td1VeHHhhb4?si=LIHrNRlIIVjZfMmS

**Reply by ghost** [reply]
lol , his it more universal but harder, Dns access/once from flags is restrictred for majority of people.

**Reply by no-name-6655** [reply]
Why is power washing a problem? And most people don't have access to js execution.

**Comment by akabutnicer** [comment]
also this is patched on v131 and above 💀

**Reply by no-name-6655** [reply]
Yes, google devs read these posts. People should probably start posting exploits in a private discord server instead.

**Reply by Luigi-Master** [reply]
> Yes, google devs read these posts. People should probably start posting exploits in a private discord server instead.

That makes sense, I've been wondering why these exploits get patched so fast

**Reply by ghost** [reply]
Its been so oblivious.

**Reply by SilverFal** [reply]
There is a way you can still do it there's a video on youtube that shows you how to block filters. It works for gogurdian but I'm not sure with other filters. The site is called nestdns.io it can create firewalls to block things like ads or web filters or other sites.

**Comment by ghost** [comment]
#relocation

**Reply by akabutnicer** [reply]
i like this

**Reply by SlitheringLizard1** [reply]
#1532

**Reply by S-PScripts** [reply]
What? They both already moved and aka is literally the owner of the discordserver

**Comment by SlitheringLizard1** [comment]
you can get a captive portal on ANY wifi if you are fast enough to change your wifi to manual dns setup or whatever and change everything to 0.0.0.0
it should open the captive portal popup
do it quackly

**Reply by S-PScripts** [reply]
what was the reasoning behind commenting this

**Reply by SlitheringLizard1** [reply]
just sharing what I found while waiting for an acc

**Reply by ghost** [reply]
you aren't the first one to discover a captive portal exploit, this has existed for a long time.

**Comment by jayden3425-gif** [comment]
I can't see as guest

**Comment by 2pro12342** [comment]
@no-name-6655 this works for kiosk apps too and it's unpatched as of v143

**Reply by SilverFal** [reply]
Yeah they banned phones at my high school and kids come to me because I constantly find ways to bypass filters.

**Reply by 2pro12342** [reply]
same
